### PR TITLE
Website + Documentation updates for mfem-4.0

### PR DESCRIPTION
**Pull request by tzanio**

### CODE REVIEW

File: src/examples.md [hunk 1]
```diff
@@ -243,7 +243,7 @@ refinement loop. The problem being solved is again the Laplace
 equation $$-\Delta u = 1$$ with homogeneous Dirichlet boundary
 conditions. The problem is solved on a sequence of meshes which
 are locally refined in a conforming (triangles, tetrahedrons)
```
Comment (mlstowell) [inline src/examples.md]: Can we also change "tetrahedrons" to "tetrahedra"?  Both are apparently correct but the later would match "hexahedra" later in the sentence.

File: src/examples.md [hunk 2]
```diff
@@ -642,6 +642,88 @@ We recommend viewing examples 2, 5 and 10 before viewing this example._
...
+  $$H = \frac{1}{2}\left[ \frac{p^2}{m} + k \left( 1 - \frac{q^2}{8} \right) q^2 \right]$$
+
+In all cases these Hamiltonians are shifted by constant values so that the
+energy will remain positive. The mean and standard deviation of the computed
+energies at each time step are displayed upon completion.
+
+When run in parallel, each processor integrates the same Hamiltonian
+system but starting from different initial conditions.
+
+See the [Maxwell](#maxwell-miniapp-transient-full-wave-electromagnetics)
```
Comment (mlstowell) [inline src/examples.md]: This example might seem unrelated to standard finite element techniques so it may be good to point out one connection in this manner.

File: src/meshing.md [hunk 3]
```diff
@@ -52,6 +52,30 @@ the mobius-strip miniapp.
...
+This miniapp creates higher dimensional meshes from lower dimensional meshes
+by extrusion.
+
+![](img/examples/extruded-star.png)
+
+Simple coordinate transformations can also be applied if desired.  The initial
+mesh can be 1D or 2D. 1D meshes can be extruded in the y-direction first and
+then in the z-direction.  2D meshes can be triangular, quadrilateral, or
+contain both element types.
+
```
Comment (pazner) [inline src/meshing.md]: Do we want to add an example of the low-order refinement here? If so, I can write something up

**Comment by pazner** [comment]
The changes all look good to me. A couple things:

- There are issues with the navigation links using MkDocs 1.0.4 (I don't have the suggested version of 0.17.4 installed on my machine). This is an easy fix, but I don't want to break compatibility with the old version
- Do we want to add 4.0rc1 to the news section on the main page?
- The `team.md` file is unused. Should it be removed?
- Maybe add something about LOR (see my comment on `meshing.md`)?

File: src/examples.md (hunk 1 above)
Comment (tzanio) [reply src/examples.md]: I'm okay with this, but can you change it consistently everywhere (including the code)?

**Comment by tzanio** [comment]
@mfem/developers -- do you have any new cool images you'd like to share on the Gallery page?
